test/tools/unified-spec-runner/schema.ts

```diff
@@ -95,15 +96,15 @@ export interface UnifiedSuite {
   tests: Test[];
   _yamlAnchors?: Document;
 }
-export const TopologyType = Object.freeze({
+export const TopologyId = Object.freeze({
```

Comment:
Why are we making this equal `TopologyID`?

Reply:
It's a pattern we use when we want to create a type that is an enum without actually using Typescript's builtin enums. The main reason we don't use Typescript's builtin enums is that they were invented by Typescript don't map cleanly onto any existing Javascript construct. (@nbbeeken, @baileympearson correct me if I'm wrong here) 

Also doing it this way frees up the actual `TopologyType` "enum" we have in the driver for ease of use in this file.

For another example of this pattern, see how the driver's `TopologyType` "enum" is defined [here](https://github.com/mongodb/node-mongodb-native/blob/eece8c1b9fcd0aa7c12722f4fe4e50e2123102b8/src/sdam/common.ts#L17-L27)

Reply:
Bookkeeping:
Discussed this thread with Warren over slack specifically clarifying that I'm wondering about the naming choice not the enum. And he will update the name to something else.